### DISCUSSION TITLE
[Broken Link]  Learn More About Projects Documentation

### DISCUSSION THREAD

**Discussion by technologirl** [community/community#27328] [Projects and Issues] [answered]
The URL to "[Learn More about (the) Projects](https://docs.github.com/issues/trying-out-the-new-projects-experience)" feature, which is supposed to lead to documentation, instead navigates to a broken link (404 page).

ETA: This also links from the "Learn More" button on the pop-up window that appears when first navigating to the Projects tab.

[https://docs.github.com/issues/trying-out-the-new-projects-experience](https://docs.github.com/issues/trying-out-the-new-projects-experience)

(I don't know how to add category tags to this discussion)

**Comment by radar-blip** [comment, marked as answer]
@technologirl Its working now.

**Reply by technologirl** [reply]
Thank you!
